src/annotator/anchoring/pdf.js

```diff
@@ -568,17 +568,14 @@ export async function anchor(root, selectors) {
 }
 
 /**
- * Convert a DOM Range object into a set of selectors.
+ * Prepare a DOM range for generating selectors and find the containing text layer.
```

Comment:
I haven't written too much in the description about the return value, because this is a private helper in this module and if I did I'd largely be restating comments from the implementation.